dotnet-desktop-guide/net/winforms/whats-new/net60.md

```diff
@@ -0,0 +1,202 @@
+{
+    [STAThread]
+    static void Main()
+    {
+        Application.EnableVisualStyles();
+        Application.SetCompatibleTextRenderingDefault(false);
+        Application.Run(new Form1());
+    }
+}
+```
+
+In .NET 6, these templates have been modified to use the new bootstrap system, invoked by the `ApplicationConfiguration.Initialize` method.
+
+```csharp
+class Program
+{
+    [STAThread]
+    static void Main()
+    {
+        ApplicationConfiguration.Initialize();
+        Application.Run(new Form1());
+    }
+}
+```
+
+This method is automatically generated at compile time and contains the code to configure Windows Forms. The project file can control these settings now too, and you can avoid configuring it in code. For example, the generated method looks similar to the following code:
+
+```csharp
+public static void Initialize()
+{
+    Application.EnableVisualStyles();
+    Application.SetCompatibleTextRenderingDefault(false);
+    Application.SetHighDpiMode(HighDpiMode.SystemAware);
+}
+```
+
+## Project-level application settings
+
+## Change the default font
+
+The default font used by Windows Forms, represented by the read-only [`Control.DefaultFont`](https://docs.microsoft.com/dotnet/api/system.windows.forms.control.defaultfont?view=windowsdesktop-6.0) property, can now be changed.
+
+Windows Forms on .NET Core 3.0 introduced a new default font for Windows Forms: **Segoe UI, 9pt**. This font better aligned to the [Windows user experience (UX) guidelines](https://docs.microsoft.com/windows/win32/uxguide/vis-fonts#fonts-and-colors). However, .NET Framework uses **Microsoft Sans Serif, 8.25pt** as the default font. This change made it harder for some customers to migrate their large applications that utilized a pixel-perfect layout from .NET Framework to .NET. The only way to change the font for the whole application was to edit every form in the project, setting the <xref:System.Windows.Forms.Control.Font> property to an alternate font.
+
+The default font can now be set in two ways:
+
+- Call the the <xref:System.Windows.Forms.Application.SetDefaultFont%2A?displayProperty=nameWithType> API in the [application bootstrap](#new-application-bootstrap) code:
+
+  ```csharp
+  class Program
+  {
+      [STAThread]
+      static void Main()
+      {
+          ApplicationConfiguration.Initialize();
+          Application.SetDefaultFont(new Font(new FontFamily("Microsoft Sans Serif"), 8.25f));
+          Application.Run(new Form1());
+      }
+  }
+  ```
+
+  \- or -
+
+- Set the default font in the project file:
+
+  ```xml
+  <Project Sdk="Microsoft.NET.Sdk">
+  
+    <!-- other settings -->
+  
+    <PropertyGroup>
+      <ApplicationDefaultFont>Microsoft Sans Serif, 8.25pt</ApplicationDefaultFont>
+    </PropertyGroup>
+  
+  </Project>
+  ```
+
+## Visual Studio designer improvements
+
+TODO: Text
+TODO: Image comparing differences
+
+## New API
+
+- <xref:System.Windows.Forms.Application.SetDefaultFont?displayProperty=fullName>
```

Comment:
For some reason this isn't rendering correctly
![image](https://user-images.githubusercontent.com/4403806/141252923-bdd67272-af4a-40dc-9239-4f36e9eb4cea.png)